src/lev-top-nav/index.js

```diff
@@ -53,7 +57,8 @@ const LevTopNav = props => {
       break;
   }
 
-  const IconTitle = <TopNav.IconTitle icon={<Logo width={logoWidth} height={logoHeight} />}>{props.companyText}</TopNav.IconTitle>;
+  const IconTitle = <TopNav.IconTitle
+    icon={<Logo width={logoWidth} height={logoHeight}/>}>{props.companyText}</TopNav.IconTitle>;
```

Comment:
This is no barrier to merging but...

I'd suggest this could be cleaned up a bit by putting `{props.companyText}` on its own line.

Reply:
Done